browser/sources.gni

```diff
@@ -329,6 +330,7 @@ brave_chrome_browser_defines = []
 
 brave_chrome_browser_sources += brave_browser_autocomplete_sources
 brave_chrome_browser_sources += brave_browser_binance_sources
+brave_chrome_browser_sources += brave_browser_brave_vpn_sources
```

Comment:
Curious we will add all new files/deps here instead of creating our new targets even if there is no circular? @iefremov @bridiver

Reply:
As far as I'm aware there is no consensus (at least, with myself :)), but the current approach is having source.gni only for circular dependencies on the Big Chrome